packages/eslint-config-react-app/base.js

```diff
@@ -29,8 +29,14 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
-    ecmaFeatures: {
-      jsx: true,
+    requireConfigFile: false,
+    // This option should be aligned to the babel-loader config
+    // applied for app sources /\.(js|mjs|jsx|ts|tsx)$/
+
+    // If a Babel plugin does not introduce a syntax extension, it is
+    // okay to not include it here since it is no-op for parser
+    babelOptions: {
+      presets: [require.resolve('babel-preset-react-app')],
```

Comment:
We use `babel-preset-react-app/dependencies` preset for node_modules. I assume that from the syntax level, `babel-preset-react-app` is always a superset to `/depedencies`.